configs/swarm/docker-compose.yml

```diff
@@ -1,46 +1,60 @@
-# This file is generated by iroha_swarm.
-# Do not edit it manually.
+# target/release/iroha_swarm -p 4 -s xyzzy -c ./configs/swarm -i hyperledger/iroha:dev -o ./configs/swarm/docker-compose.yml -HF
+# 
+# This configuration has been generated by `iroha_swarm` using the above settings.
+# You should not edit this manually.
 
-version: '3.8'
 services:
+  image_provider:
```

Comment:
Is image provider needed when irohad services have `image`, not `build`?

Reply:
Not really, I see how this can be simplified.